analysis/pf_dataset.py

```diff
@@ -13,6 +15,15 @@
     "combined_pf_service": ["1577041000000109", "983341000000102"],
 }
 
+def get_pf_consultation_ids():
+    pharmacy_first_ids = clinical_events.where(
+    clinical_events.snomedct_code.is_in(pharmacy_first_event_codes["combined_pf_service"])
+    ).consultation_id
+    return pharmacy_first_ids
```

Comment:
I think this function should have one or two more arguments, because at the moment it depends on too many external things being set up properly, we could solve that for example by making the function more general and providing two arguments: 

```suggestion
def get_consultation_ids_from_events(events, codelist):
    selected_consultation_ids = events.where(
    events.snomedct_code.is_in(codelist)
    ).consultation_id
    return pharmacy_first_ids
```

Reply:
So the events would be clinical_events, and the codelist would be? Not sure how I would call the codelist value, as pharmacy_first_event_codes["combined_pf_service"] is not on codelist.py, its a dictionary in the pf_dataset.py file.

Reply:
I'd do something like this, I also left a suggestion how to implement it in `analysis/pf_dataset_definition.py` 

```py
pharmacy_first_ids = get_consultation_ids_from_events(clinical_events, pharmacy_first_event_codes["combined_pf_service"])
```